ui/ceval/src/stockfishProtocol.js

```diff
@@ -1,11 +1,17 @@
 var m = require('mithril');
 
+var EVAL_REGEX = new RegExp(''
+  + /^info depth (\d+) seldepth \d+ multipv (\d+) /.source
+  + /score (cp|mate) ([-\d]+) /.source
+  + /(?:(upper|upper)bound )?nodes (\d+) nps (\d+) /.source
+  + /(?:hashfull \d+ )?tbhits \d+ time (\d+) /.source
+  + /pv ?(.*)/.source);
```

Comment:
this is changing in follow up to `pv(.*)` and then lower `pv = matches[9].trim()`.

In the sf print function, an empty PV means no space, so I am defensively coding the parser. Not sure if it's possible for SF to have an empty PV, didn't investigate that.